pipelines/environments/mldesigner-minimal-2.0-ubuntu20.04-py38-cpu copy/environment.yaml

```diff
@@ -0,0 +1,12 @@
+image:
+  name: azureml/curated/mldesigner-minimal
```

Comment:
Is this in the MCR config? I only see azureml/curated/mldesigner there right now.

Reply:
We have a PR working on this: microsoft/mcr#2383

Reply:
We'll need to wait for the MCR PR to merge before this PR can merge.